.pre-commit-config.yaml

```diff
@@ -39,7 +39,7 @@ repos:
       - id: pyupgrade
         args: [--py37-plus]
   - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
+    rev: 5.12.0
```

Comment:
This probably should have been a separate PR

Reply:
Can do for next time. Sounds good. Was just causing an issue with ci/cd pipeline